src/examples/java/io/nats/examples/autobench/JsPubAsyncRoundsBenchmark.java

```diff
@@ -70,7 +70,7 @@ public void execute(Options connectOptions) throws InterruptedException {
                     for (long l = 0; l < thisRound; l++) {
                         futures.add(js.publishAsync(subject, payload));
                     }
-                    while (futures.size() > 0) {
+                    while (!futures.isEmpty()) {
```

Comment:
Changing code just to change code is unnecessary. I get that this is the preferred way, my ide tells me so all the time, it's probably more readable / expressive, but it's actually more expensive for most of the collections we use since `isEmpty` asks `size() == 0`